data/net/minecraft/util/Mth.mapping

```diff
@@ -238,12 +292,47 @@ CLASS net/minecraft/util/Mth
 			COMMENT Start value for the lerp
 		ARG 2 end
 			COMMENT End value for the lerp
+	METHOD lerp2 (DDDDDD)D
+		ARG 0 delta
+		ARG 2 delta2
+		ARG 4 start
+		ARG 6 end
+		ARG 8 start2
+		ARG 10 end2
+	METHOD lerp3 (DDDDDDDDDDD)D
+		ARG 0 delta
+		ARG 2 delta2
+		ARG 4 delta3
+		ARG 6 start
+		ARG 8 end
+		ARG 10 start2
+		ARG 12 end2
+		ARG 14 start3
+		ARG 16 end3
+		ARG 18 start4
+		ARG 20 end4
```

Comment:
I would annotate 1,2,3,etc. when dealing with params with the same name.

Additionally, start and end doesn't make much sense since explanations of where the multi-interpolation is happening is unclear.